### ISSUE TITLE
MediaElch crashes upon searching for movie details  error 4 in libQt5Widgets.so.5.15.1

### ISSUE THREAD

**Issue by aljazzair** [Komet/MediaElch#1060]
**Describe the bug**
MediaElch crashes upon searching for movie details

**To Reproduce**
1. Install MedialElch on Manjaro-Arch - latest version 2.6.6
2.  Launch MedialElch and select directory with movies from the settings, click save
3. Reload library from the UI
4. Select a movie on the left panel
5. Right Click "Load Information"
6. Movie is detected by scraper, click on the result to load movie information
7. Immediate Crash with following details from log (using journalctl -grep lch)

```
Nov 25 13:31:53 mycomputer kernel: MediaElch[1121587]: segfault at 129 ip 00007f22bd28fd57 sp 00007ffefd871e90 error 4 in libQt5Widgets.so.5.15.1[7f22bd259000+3b0000]
Nov 25 13:31:53 mycomputer kernel: audit: type=1701 audit(1606307513.678:818): auid=2107 uid=2107 gid=2020 ses=1 pid=1121587 comm="MediaElch" exe="/usr/bin/MediaElch" sig=11 res=1
Nov 25 13:31:54 mycomputer systemd[1905]: app-gnome-MediaElch-1121587.scope: Succeeded.
Nov 25 13:31:54 mycomputer systemd-coredump[1123372]: Process 1121587 (MediaElch) of user 2107 dumped core.
                                                   #16 0x000055f84f6d2dc2 main (MediaElch + 0xa7dc2)
                                                   #18 0x000055f84f6d3c2e n/a (MediaElch + 0xa8c2e)
```

**Expected behavior**
No Crash

**MediaElch Version:**
 - [ ] 2.6.6 stable

**Operating System:**
 - Linux server 5.9.10-1-MANJARO #1 SMP PREEMPT Sun Nov 22 11:25:19 UTC 2020 x86_64 GNU/Linux

**Additional context**
This is a fresh install of Manjaro/Arch.

**Comment by aljazzair** [comment]
Looks like this issue is similar to #1050 - could be duplicate.

**Comment by bugwelle** [comment]
Thanks for reporting. Yes, there are multiple duplicates of this issue. I'm setting up a VM with Manjaro to test this our right now. :-)

**Comment by bugwelle** [comment]
Can't reproduce it yet in a virtual machine with Manjaro KDE desktop. Which desktop environment are you using? :-)

**Comment by bugwelle** [comment]
I was able to reproduce this on Manjaro with XFCE :tada: :disappointed:

**Comment by aljazzair** [comment]
Hello
Thank you for the follow up. My DE is Gnome. 
My CPU is a single Intel 8700k, it has 6 cores, 12 threads.

**Comment by bugwelle** [comment]
Hi,

I just wanted to inform you that I was able to reproduce this issue on Manjaro XFCE and that the bug will be fixed by #1078.
Turns out that this bug only happened with Kvantum installed which is why I wasn't able to reproduce this bug (I used Manjaro KDE for testing before...).

Regards,
Andre

**Comment by bugwelle** [comment]
#1078  is now merged. If you compile MediaElch yourself, check out the latest master. If not you will have to wait for the next release.

If this issue wasn't fixed for you, please let me know.
